javaparser-symbol-solver-core/src/main/java/com/github/javaparser/symbolsolver/javaparsermodel/declarations/JavaParserClassDeclaration.java

```diff
@@ -290,16 +290,21 @@ public boolean isTypeParameter() {
     @Override
     public List<ResolvedReferenceType> getAncestors() {
         List<ResolvedReferenceType> ancestors = new ArrayList<>();
-        ResolvedReferenceType superclass = getSuperClass();
-        if (superclass != null) {
-            ancestors.add(superclass);
-        }
-        if (wrappedNode.getImplementedTypes() != null) {
-            for (ClassOrInterfaceType implemented : wrappedNode.getImplementedTypes()) {
-                ResolvedReferenceType ancestor = toReferenceType(implemented);
-                ancestors.add(ancestor);
+
```

Comment:
I agree with the solution, which is necessary if we want to support parsing Object itself. I never thought that was possible.